module/data/activity/check-data.mjs

```diff
@@ -61,6 +61,15 @@ export default class CheckActivityData extends BaseActivityData {
   /*  Data Migrations                             */
   /* -------------------------------------------- */
 
+  /** @override */
+  static migrateData(source) {
```

Comment:
We shouldn't need to migrate anything. This data model changed during development, not between releases.